src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -161,21 +161,33 @@ function OrganizationDashboard(): JSX.Element {
                   </Link>
                 </Col>
                 <Col sm={4} className="mb-5">
-                  <div className={`card ${styles.cardContainer}`}>
-                    <div className="card-body">
-                      <div className="text-center mb-3">
-                        <i
-                          className={`fas fa-user ${styles.dashboardIcon}`}
-                        ></i>
-                      </div>
-                      <div className="text-center">
-                        <p className={styles.counterNumber}>
-                          {data.organizations[0].admins.length}
-                        </p>
-                        <p className={styles.counterHead}>{t('admins')}</p>
+                  <Link
+                    className={`card ${styles.cardContainer}`}
+                    to={`${targets
+                      .filter((target: any) => {
+                        const { name } = target;
+                        return name == 'People';
+                      })
+                      .map((target: any) => {
+                        return target.url;
+                      })}`}
+                  >
+                    <div className={`card ${styles.cardContainer}`}>
+                      <div className="card-body">
+                        <div className="text-center mb-3">
+                          <i
+                            className={`fas fa-user ${styles.dashboardIcon}`}
+                          ></i>
+                        </div>
+                        <div className="text-center">
+                          <p className={styles.counterNumber}>
+                            {data.organizations[0].admins.length}
```

Comment:
Does this mean it will always display only the admins for the very first organization and not show data for any others?

Reply:
Line 184 will show the number of people having the status of 'admin' or 'superadmin' **under that organization only.**
Suppose we are checking the dashboard of org 'A'. Which have 1 admin. It will display 1.
we are using `organizations[0].admin.length` because of the format in which `data` object is returned in the query which is at top of the component. Organization[0] does not mean the frist organization in the database